src/adr/0018-hexagonal-architecture.md

```diff
@@ -0,0 +1,44 @@
+# 17. FileMailQueue deprecation
```

Comment:
it's not the right content :)

Reply:
sorry it's a mistake while fixing checkstyle. I'll remove it.

Reply:
Anyway, won't it deserve a separate ADR pr ;-)?